tests/lib/rules/implicit-arrow-linebreak.js

```diff
@@ -132,6 +199,317 @@ ruleTester.run("implicit-arrow-linebreak", rule, {
                     )
             `,
             errors: [UNEXPECTED_LINEBREAK]
+        }, {
+            code: `
+                (foo) =>
+                  // test comment
+                  bar
+            `,
+            output: `
+                // test comment
+                (foo) => bar
+            `,
+            errors: [UNEXPECTED_LINEBREAK]
+        }, {
+            code: `
+                const foo = () =>
+                // comment
+                []
+            `,
+            output: `
+                // comment
+                const foo = () => []
+            `,
+            errors: [UNEXPECTED_LINEBREAK]
+        }, {
+            code: `
+                (foo) =>
+                    (
+                    //comment
+                        bar
+                    )
+            `,
+            output: `
+                (foo) => (
+                    //comment
+                        bar
+                    )
+            `,
+            errors: [UNEXPECTED_LINEBREAK]
+        }, {
+            code: `
+                (foo) =>
+                    (
+                        bar
+                    //comment
+                    )
+            
+            `,
+            output: `
+                (foo) => (
+                        bar
+                    //comment
+                    )
+            
+            `,
+            errors: [UNEXPECTED_LINEBREAK]
+
+        }, {
+            code: `(foo) =>
+                    // comment
+                    // another comment
+                        bar`,
+            output: "// comment\n// another comment\n(foo) => bar",
```

Comment:
Nit: It would really help to see this represented as a template since the code example is also a template-- that way it's easier to see the differences visually.

Reply:
@platinumazure Thanks for the comment! I applied the requested change to this test case and the one below.

Reply:
Thanks @peanutenthusiast, much appreciated! I'll review again tonight.